src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -301,7 +317,8 @@ public void TimeOut_SetThenGet_ValuesMatch(Uri remoteServer)
                 var sw = Stopwatch.StartNew();
                 WebException exception = Assert.Throws<WebException>(() =>
                 {
-                    var response = (HttpWebResponse)request.GetResponse();
+                    var response = request.GetResponse();
+                    response.Dispose();
```

Comment:
I don't understand this change.  Isn't it GetResponse that's throwing the exception?  If so there won't ever be anything to Dispose.